packages/web-components/src/components/ic-date-input/ic-date-input.tsx

```diff
@@ -1189,8 +1400,37 @@ export class DateInput {
     }
   };
 
+  private handleClear = () => {
+    this.inputsInOrder.forEach((input) => {
+      input.classList.remove(this.FIT_TO_VALUE);
+      this.setInputValue(input, true);
+      this.setPreventInput(input, false);
+    });
+    this.isDateSetFromKeyboardEvent = false;
+    this.value = null;
+    this.setValidationMessage();
+    this.handleDateChange(true);
+
+    setTimeout(() => {
+      this.dayInputEl.focus();
+    }, 100);
```

Comment:
Is `setTimout` still required? Also, what if the date format is YYYY/MM/DD or MM/DD/YYYY? The focus will go to the incorrect input. Can use `this.inputsInOrder()[0]` or something like that